lib/cluster.js

```diff
@@ -694,8 +696,10 @@ function workerInit() {
     if (!this.isConnected()) {
       process.exit(0);
     } else {
-      send({ act: 'exitedAfterDisconnect' }, () => process.disconnect());
-      process.once('disconnect', () => process.exit(0));
+      const processDisconnect = () => process.disconnect();
```

Comment:
Yes, I intended to mention you. You seem tuned into what is going on, I thought maybe the CTC had made a decision somewhere that this was the way to go that you would know about.